Notify/Notify/Notify/AppShell.xaml.cs

```diff
@@ -203,30 +224,29 @@ private List<Notification> getAllArrivedLocationNotifications(List<string> desti
                 .FindAll(
                     notification =>
                     {
-                        bool isLocationNotification = notification.Type is NotificationType.Location;
+                        bool isCurrentType = notification.Type is NotificationType.Location || notification.Type is NotificationType.Dynamic;
```

Comment:
I am not sure I understood the name of the variable, currentType of what?

Reply:
It supposed to be "isCorrectType".
Probably caused by mistyping.
Although I think that "isRelevantType" is a better name after all. Changing it now.

Reply:
`isRelevantType` is good